templates/shrine/src/shrine-detail.html

```diff
@@ -243,7 +254,8 @@
           <h1>[[item.title]]</h1>
           <p>[[item.description]]</p>
           <div class="quantitySelector">
-            <select>
+            <label class="sr-only" for="quantity">Number of items:</label>
+            <select id="quantity">
```

Comment:
I think in this case is fine to just use `aria-label` because:
1. You gain the same accessibility as provided by `label`.
2. You save a DOM node.
3. In this PR, you are hiding the `<label>`, so there isn't really a good use case for an element here.